modules/local/samplesheet_check.nf

```diff
@@ -1,8 +1,8 @@
 process SAMPLESHEET_CHECK {
     tag "$samplesheet"
-    label 'process_low'
+    label 'process_single'
 
-    conda (params.enable_conda ? "conda-forge::python=3.8.3" : null)
+    conda "conda-forge::python=3.8.3"
```

Comment:
Need to do the same in https://github.com/nf-core/isoseq/blob/dev/modules/local/gstama/filelist/main.nf

Reply:
Updated, Thanks for pointing this.